src/System.Runtime.Loader/tests/System.Runtime.Loader.Noop.Assembly/project.json

```diff
@@ -1,17 +1,9 @@
 {
   "dependencies": {
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-24102-00",
     "System.Runtime": "4.1.0-rc3-24102-00"
   },
   "frameworks": {
     "netstandard1.0": {}
-  },
-  "runtimes": {
-    "win7-x86": {},
-    "win7-x64": {},
-    "ubuntu.14.04-x64": {},
-    "osx.10.10-x64": {},
-    "centos.7-x64": {},
-    "rhel.7-x64": {},
-    "debian.8.2-x64": {}
```

Comment:
This isn't actually a deploy-able test.  It's just a dummy assembly.  That's why we're OK not having the runtimes.  The actual test project retains the runtimes.